packages/toolpad-core/src/DashboardLayout/DashboardLayout.test.tsx

```diff
@@ -230,7 +230,7 @@ describe('DashboardLayout', () => {
 
     const { rerender } = render(<AppWithPathname pathname="/dashboard" />);
 
-    const navigation = screen.getByRole('navigation');
+    const navigation = screen.getAllByRole('navigation').at(-1)!;
```

Comment:
Might want to label these instead, as per https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role#multiple_landmarks

Usually if you find yourself crafting complex queries, it's a sign you're lacking accessibility.

Reply:
Ah cool, there should never actually be multiple `navs` in the page except in these headless browser tests, but anyway doesn't hurt to add these.

Reply:
The way I look at it is that if it's hard to navigate the accessibility tree in the tests, it could be an indicator that it's going to be hard for screenreaders as well.